### ISSUE TITLE
Segmentation Faults

### ISSUE THREAD

**Issue by madarax64** [isakkarlsson/wildboar#8]
Hello,
I installed wildboar on Ubuntu 18.04 LTS, with Python 3.6.8 using pip. The examples run perfectly fine, but whenever I try to run it on my data I get a segmentation fault. How can I resolve this?

Thanks

**Comment by isakkarlsson** [comment]
Hi!

Thanks for reporting! To resolve it I need more information about the data and the code you try to run.

Isak

**Comment by madarax64** [comment]
Hello,
My apologies, I think I've figured it out - my data is a univariate timeseries, which was shaped as `[n_samples, 431, 1]` (i.e n_samples, with 431 univariate samples). Going through the issues I noticed that this inadvertently indicated that my timeseries was multivariate and had 431 dimensions...reshaping this to `[n_samples, 431]` fixed the problem right up.

On a side note, I think its necessary to add that for the ShapeletForestClassifier, you actually don't seem to need to reshape/flatten the data? It seems to work directly on data with shape `[n_samples, n_dims, n_timesteps]` ?

In any case, I will now close this issue. Apologies for the gaffe once again..
